aosp_diff/caas_cvmb/frameworks/av/0002-Resolve-the-issue-of-OMX.Intel.sw_vd.h264-5-not-bein.patch

```diff
@@ -0,0 +1,56 @@
+From 04d59eb510fcba53485ed6bb8da4c4371f9dcd8f Mon Sep 17 00:00:00 2001
+From: "Wan, Hao" <haox.wan@intel.com>
+Date: Thu, 11 Apr 2024 11:39:15 +0000
+Subject: [PATCH] Resolve the issue of OMX.Intel.sw_vd.h264(5) not being loaded
+
+Signed-off-by: Wan, Hao <haox.wan@intel.com>
+---
+ media/libstagefright/omx/OMXStore.cpp | 7 ++++++-
+ 1 file changed, 6 insertions(+), 1 deletion(-)
+
+diff --git a/media/libstagefright/omx/OMXStore.cpp b/media/libstagefright/omx/OMXStore.cpp
+index 4827d9e056..dec0555f41 100644
+--- a/media/libstagefright/omx/OMXStore.cpp
++++ b/media/libstagefright/omx/OMXStore.cpp
+@@ -110,6 +110,7 @@ static int getFirstApiLevel() {
+     return android::base::GetIntProperty("ro.product.first_api_level", __ANDROID_API_T__);
+ }
+ 
++#if 0
+ static bool isTV() {
+     static const bool kIsTv = []() {
+         std::string characteristics = android::base::GetProperty("ro.build.characteristics", "");
+@@ -123,11 +124,13 @@ static bool isTV() {
+     }();
+     return kIsTv;
+ }
++#endif
+ 
+ void OMXStore::addPlugin(OMXPluginBase *plugin) {
+     Mutex::Autolock autoLock(mLock);
+-
++#if 0
```

Comment:
Instead of #if 0

Shall we define some thing like
#define TARGET_CHECK_TV 0

and use that

#if TARGET_CHECK_TV